hashes/zkevm-keccak/src/keccak_packed_multi.rs

```diff
@@ -1546,6 +1510,43 @@ impl<F: Field> KeccakCircuitConfig<F> {
         }
     }
 
+    /// Assigns the cells in `KeccakRow` to `region`.
```

Comment:
Could you explain the extra `with_flags` part here as well?

Reply:
Explained